examples/dynamic_penguin.ipynb

```diff
@@ -0,0 +1,231 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "pygame 1.9.6\n",
+      "Hello from the pygame community. https://www.pygame.org/contribute.html\n"
+     ]
+    }
+   ],
+   "source": [
+    "import asyncio\n",
+    "from twisted.internet import asyncioreactor\n",
+    "asyncioreactor.install(asyncio.get_event_loop())\n",
+    "from twisted.internet import reactor\n",
+    "import ppb\n",
+    "import txppb"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "That's it.\n",
+    "Now,\n",
+    "I like penguins,\n",
+    "so I want to have a penguin character.\n",
+    "We have a file called\n",
+    "`penguin.png`\n",
+    "with a little cute round penguin.\n",
+    "\n",
+    "PPB uses the class name to find the right image."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class Penguin(ppb.BaseSprite):\n",
+    "    pass"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Well, now we have a Penguin class,\n",
+    "boring though it may be,\n",
+    "we need to make an\n",
+    "*instance*\n",
+    "of it.\n",
+    "\n",
+    "PPB thinks in terms of\n",
+    "\"scenes\",\n",
+    "so we need to set up a scene,\n",
+    "with one penguin in it.\n",
+    "\n",
+    "We give it a tag:\n",
+    "the tag will be useful later."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def setup(reactor, scene):\n",
+    "    scene.add(Penguin(pos=(0, 0)), tags=['penguin'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we need to run the\n",
+    "\"game loop\".\n",
+    "This is the loop that,\n",
+    "eventually,\n",
+    "will look at user actions\n",
+    "and will do stuff to the game.\n",
+    "\n",
+    "Right now,\n",
+    "it will just show a penguin.\n",
+    "In the middle of the screen.\n",
+    "Just sitting there."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<Deferred at 0x7f4bb80d2f90>"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "d = txppb.run(reactor, setup)\n",
+    "d.addBoth(print)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This is really boring, right?\n",
+    "The penguin is just sitting there.\n",
+    "\n",
+    "Let's make it do something.\n",
+    "Maybe go up?\n",
+    "But slowly, so we can keep an eye on it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "SPEED = 0.5\n",
```

Comment:
Consider making this a class attribute ala

```suggestion
    "Penguin.speed = 0.5\n",
```

You can then use self.speed inside the new handler. (Not a strict suggestion, but closer to the best practices with ppb.)

Reply:
Done